apps/namadillo/src/integrations/utils.ts

```diff
@@ -44,6 +44,7 @@ export const findRegistryByChainId = (
 };
 
 export const getChainFromAddress = (address: string): Chain | undefined => {
+  const chains = getAvailableChains();
```

Comment:
Stride wasn't selectable b/c here I was using the `chains` from the registry and it was getting matched with the `stratos` chain instead of `stride`.